docs/csharp/language-reference/keywords/var.md

```diff
@@ -32,7 +32,7 @@ translation.priority.ht:
   - "zh-tw"
 ---
 # var (C# Reference)
-Beginning in Visual C# 3.0, variables that are declared at method scope can have an implicit type `var`. An implicitly typed local variable is strongly typed just as if you had declared the type yourself, but the compiler determines the type. The following two declarations of `i` are functionally equivalent:  
+Beginning in Visual C# 3.0, variables that are declared at method scope can have an implicit "type" `var`. An implicitly typed local variable is strongly typed just as if you had declared the type yourself, but the compiler determines the type. The following two declarations of `i` are functionally equivalent:  
```

Comment:
Agree. Should we get rid of mentioning the same abstraction in ["Implicitly typed local variables"](https://github.com/dotnet/docs/blob/master/docs/csharp/programming-guide/classes-and-structs/implicitly-typed-local-variables.md) article? The very first sentences say:

> Local variables can be given an inferred "type" of `var` instead of an explicit type. The `var` keyword instructs the compiler to infer the type of the variable from the expression on the right side of the initialization statement.

We can change it something like:

> Local variables can be declared without giving an explicit type. The `var` keyword instructs the compiler to infer the type of the variable from the expression on the right side of the initialization statement.